### ISSUE TITLE
Separate the extra standard deduction amounts for aged and blind

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-us#758]
The extra standard deductions for aged and blind filers are specified in https://www.law.cornell.edu/uscode/text/26/63#f as separate parameters in sections 1 and 2.

However, we implement it as a single parameter, since they are the same value. Both to stay closer to the law and for clarity (e.g., I was initially unsure whether people who are both aged and blind get twice the extra amount), I'd suggest we separate them.

**Comment by fedderw** [comment]
Yeah they should definitely be separate - age by itself won't get you Medicaid eligibility

**Comment by MaxGhenis** [comment]
@fedderw This is for federal income taxes rather than Medicaid

**Comment by fedderw** [comment]
> @fedderw This is for federal income taxes rather than Medicaid

whoops, my bad. I'm a little mixed up. I think I did mean to say this somewhere though 🤔
